.travis.yml

```diff
@@ -182,13 +182,15 @@ install:
 script:
   - ./gdal/ci/travis/${BUILD_NAME}/script.sh
 
-# notifications:
+notifications:
+  irc:
+    channels:
+       - "irc.freenode.org#gdal-activity"    
+      #- "irc.freenode.org#gdal"
+   use_notice: true
```

Comment:
I believe this line and the following one should receive an extra space for indentation to make it valid .yml

Reply:
good catch, thanks @rouault